src/main/java/com/cburch/logisim/LogisimVersion.java

```diff
@@ -103,16 +106,27 @@ public int compareTo(LogisimVersion other) {
     // means stable version, while presence of suffix indicates unstable one, so in case
     // all other values are equal, "no suffix" is considered newer.
     if (result == 0) {
-      if (this.suffix == "" && other.suffix != "") {
+      if (this.suffix.equals("") && !other.suffix.equals("")) {
         result = 1; // this one is newer
-      } else if (this.suffix != "" && other.suffix == "") {
+      } else if (!this.suffix.equals("") && other.suffix.equals("")) {
         result = -1; // this one is older
       }
     }
 
     return result;
   }
 
+  /**
+   * Returns TRUE if version is considered stable, false otherwise. Note the implementation is
+   * plaind dumb and relies on presence of version suffix (which, if not empty means non-stable
+   * release).
+   *
+   * @return
+   */
+  public boolean isStable() {
+    return "".equals("");
```

Comment:
No, you fixed the if on line 139 in the next commit. This is a return on line 127. The return still needs to be fixed.